src/Medology/Behat/CsvContext.php

```diff
@@ -118,4 +119,22 @@ private function assertNoAdditionalHeaders(array $required, array $actual, strin
             throw new Exception("CSV '$csvName' contains extra headers '" . implode("', '", $diff) . "'");
         }
     }
+
+    /**
+     * Update expected data injecting model values.
+     *
+     * @throws ReflectionException
```

Comment:
Please add missing information to this docblock.

```suggestion
     *
     * @param TableNode                 $table A table node containing the values to inject.
     * @throws ReflectionException When {the reason why the exception was thrown}
```